stellargraph/data/unsupervisedSampler.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jan 16 15:00:01 2019
+
+@author: hab031
+"""
+
+
+__all__ = ["UnsupervisedSampler"]
+
+
+import random
+import numpy as np
+
+from stellargraph.core.utils import is_real_iterable
+from stellargraph.core.graph import StellarGraphBase
+from stellargraph.data.explorer import UniformRandomWalk
+
+
+class UnsupervisedSampler:
+    """
+        This class does unsupervised samlping in a graph. 
```

Comment:
Keep comments to around 100 characters max, so they are more easily visible in editors.